tests/light/shared_files/valid-ca.crt

```diff
@@ -0,0 +1,12 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We already have certificates for server and ca in our codebase: `axosyslog/tests/light/shared_files/`.
Why do we need new certificates, are not they good (the old one)?

Reply:
The old certs are not valid and they are expired. Peer verification is key in this test.

Reply:
Thanks for the info, I did not know about it (the server cert already expired). 
In this case it is ok. Maybe in a separate PR I would remove expired cert and keep only the new